agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -146,6 +171,17 @@ public static final TemplateContext buildTemplateParameters(TemplateProvider tem
     setEventDetails(templateContext, notification);
     templateContext.put(TEMPLATE_VARIABLE_EVENT_IS_NEW, notification.getValueOwnerParameter(STORED_PARAMETER_EVENT_IS_NEW));
     templateContext.put(TEMPLATE_VARIABLE_EVENT_URL, notification.getValueOwnerParameter(STORED_PARAMETER_EVENT_URL));
+    templateContext.put(TEMPLATE_VARIABLE_EVENT_CREATOR, notification.getValueOwnerParameter(STORED_PARAMETER_EVENT_CREATOR));
+    if (notification.getValueOwnerParameter(STORED_PARAMETER_EVENT_IS_NEW).equals("false")) {
```

Comment:
use StringUtils.equals to avoid NPE on notification.getValueOwnerParameter(STORED_PARAMETER_EVENT_IS_NEW)